src/core/public/chrome/chrome_service.tsx

```diff
@@ -143,6 +150,14 @@ export class ChromeService {
     );
   }
 
+  public setup() {
+    return {
+      registerCollapsibleNavHeader: (render: CollapsibleNavHeaderRender) => {
+        this.collapsibleNavHeaderRender = render;
+      },
+    };
```

Comment:
is it ok to set collapsibleNavHeaderRender multiple times to override previous one, or we should add a check here to let it only set it once?

Reply:
You mean if different plugins called `registerCollapsibleNavHeader` with different `render`, previous one will be override?
Actually I have the same concern, but it should be fine as this will only be used by workspace for now. If in the future, this is called by different plugins, we may add the check or register the render to an array, I don't know which is expected, so just leave it like this for now.